boards.txt

```diff
@@ -11632,6 +11731,148 @@ heltec_wifi_lora_32_V2.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
+heltec_wifi_lora_V3.name=Heltec WiFi LoRa 32(V3) / Wireless shell(V3) / Wireless stick lite (V3)
```

Comment:
Should the name be `heltec_wifi_lora_32_V3` instead of `heltec_wifi_lora_V3`? To match previous board definition ( as `heltec_wifi_lora_32_V2`)? Rename it please everywhere. The variant folder name is right :)